src/components/Radio.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <fieldset class="f-radio">
+    <input
+      v-model="newResourceType.abstract"
```

Comment:
even if we do not delete this, shouldn't we rename it to something more generic?